web-console/knikubevirt/vm-details/devices-and-resources/devices-and-respurces.md

```diff
@@ -0,0 +1,40 @@
+# VM details - Devices-and-resources
+
+### Virtual machine
+
+## From VM details page
+![VM details - Overview tab - default](img/VM-2-0.jpg)
+Within the VM details > overview tab > overview section, a click on the ‘Devices & resources’ field will pop up the correlated modal.
+
+![Devices & resources modal - default](img/VM-3-0.png)
```

Comment:
should this say "on scheduled resources"?

Reply:
Yes, but I'll wait to see how the suggestion on the previous comment turns out